web/src/OverviewTable.test.tsx

```diff
@@ -377,39 +361,39 @@ describe("overview table with groups", () => {
 
   describe("display", () => {
     it("does not show the resource groups tooltip", () => {
-      expect(wrapper.find(ResourceGroupsInfoTip).length).toBe(0)
+      expect(wrapper.querySelectorAll(".MuiTooltip").length).toBe(0)
     })
 
     it("renders each label group in order", () => {
       const { labels: sortedLabels } = resources
-      const groupNames = wrapper.find(OverviewGroupName)
+      const groupNames = wrapper.querySelectorAll(OverviewGroupName)
 
       // Loop through the sorted labels (which includes every label
       // attached to a resource, but not unlabeled or tiltfile "labels")
       sortedLabels.forEach((label, idx) => {
-        const groupName = groupNames.at(idx)
-        expect(groupName.text()).toBe(label)
+        const groupName = groupNames[idx]
+        expect(groupName.textContent).toBe(label)
       })
     })
 
     // Note: the sample data generated in the test helper `nResourcesWithLabels`
     // always includes unlabeled resources
     it("renders a resource group for unlabeled resources and for Tiltfiles", () => {
-      const groupNames = wrapper.find(OverviewGroupName)
+      const groupNames = Array.from(wrapper.querySelectorAll(OverviewGroupName))
 
-      const unlabeledLabel = groupNames.filterWhere(
-        (name) => name.text() == UNLABELED_LABEL
+      const unlabeledLabel = groupNames.filter(
+        (name: any) => name.textContent == UNLABELED_LABEL
       )
       expect(unlabeledLabel.length).toBe(1)
 
-      const tiltfileLabel = groupNames.filterWhere(
-        (name) => name.text() == TILTFILE_LABEL
+      const tiltfileLabel = groupNames.filter(
+        (name: any) => name.textContent == TILTFILE_LABEL
       )
       expect(tiltfileLabel.length).toBe(1)
     })
```

Comment:
(suggestion) i believe this test could be shortened to just these `expect`s. 

```
expect(screen.findByText(UNLABLELED_LABEL)).toBeTruthy()
expect(screen.findByText(TILTFILE_LABEL)).toBeTruthy()
```

Reply:
done